### PR TITLE
Tag the F# team for review and ownership

### PR DESCRIPTION
**Pull request by BillWagner**
The F# docs had metadata that still pointed to the former language PM. Replace that with the F# compiler team and current PM.

While making those changes, update the CODEOWNERS file for the same reason.

### CODE REVIEW

File: docfx.json [hunk 1]
```diff
@@ -314,7 +314,6 @@
                 "docs/core/extensions/**/**.md": "IEvangelist",
                 "docs/core/docker/**/**.md": "IEvangelist",
                 "docs/core/native-interop/**/**.md": "jkoritzinsky",
-                "docs/core/porting/**/**.md": "cartermp",
```
Comment (BillWagner) [inline docfx.json:317]: @gewarren 

I tagged you on this one, because the up-folder default is you. Is that right, or should it be someone else?

**Review by ajcvickers — APPROVED** [review]

**Review by KathleenDollard — APPROVED** [review]
The F# changes look good to me.

I assume it is correct that Arthur is sometimes avickers and sometimes ajvickers?

**Comment by ajcvickers** [comment]
I'm avickers at Microsoft, but ajcvickers on GitHub.

File: docfx.json (hunk 1 above)
Comment (gewarren) [reply docfx.json:317]: @BillWagner I'm fine either way.

**Pull request merged**